cli/using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,28 @@
+[id="using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: using-roxctl-cli
+
+toc::[]
+
```

Comment:
@dhaus67 Should we add a new section with roxctl whoami example? if so, please can you share the command with an example, thx!
https://docs.google.com/document/d/13Tjfv3e_Mo7TvTotnN_6iuo3tLa2ZCR472FDqC6gzYQ/edit#bookmark=id.qbpc4m7r9bgt

Reply:
Yeah I think `roxctl central whoami` would be a good "getting started" thing.

The command would be: `roxctl central whoami`, do you need some sample output as well?

Reply:
@dhaus67 yes, a sample output would be helpful, thx!

Reply:
```
roxctl central whoami                                                                                                                                                                                                                                                                                                 
UserID:
	<redacted>
User name:
	<redacted>
Roles:
 APIToken creator, Admin, Analyst, Continuous Integration, Network Graph Viewer, None, Sensor Creator, Vulnerability Management Approver, Vulnerability Management Requester, Vulnerability Manager, Vulnerability Report Creator
Access:
  rw Access
  rw Administration
  rw Alert
  rw CVE
  rw Cluster
  rw Compliance
  rw Deployment
  rw DeploymentExtension
  rw Detection
  rw Image
  rw Integration
  rw K8sRole
  rw K8sRoleBinding
  rw K8sSubject
  rw Namespace
  rw NetworkGraph
  rw NetworkPolicy
  rw Node
  rw Secret
  rw ServiceAccount
  rw VulnerabilityManagementApprovals
  rw VulnerabilityManagementRequests
  rw WatchedImage
  rw WorkflowAdministration
```